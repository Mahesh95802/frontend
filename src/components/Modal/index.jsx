import React from 'react';
import PropTypes from 'prop-types';

import './Modal.css';

const Modal = ({ children }) => {
	return (
		<div className="modal">
			<div className="modal-content">
				{children}
			</div>
		</div>
	);
};

Modal.propTypes = {
	children: PropTypes.node,
};

export default Modal;