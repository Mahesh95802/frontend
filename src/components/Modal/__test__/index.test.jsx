/* eslint-disable no-undef */
import React from 'react';
import { render } from '@testing-library/react';
import Modal from '..';

describe('Modal', () => {
    it('should render correctly', () => {
        const { asFragment } = render(<Modal><p>Hii</p></Modal>);
        expect(asFragment()).toMatchSnapshot();
    });
});