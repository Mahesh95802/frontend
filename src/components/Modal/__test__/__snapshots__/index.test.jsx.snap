// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal should render correctly 1`] = `
<DocumentFragment>
  <div
    class="modal"
    data-testid="modal"
  >
    <div
      class="modal-content"
    >
      <p>
        Hii
      </p>
    </div>
  </div>
</DocumentFragment>
`;
